helix-term/src/application.rs

```diff
@@ -449,13 +449,17 @@ impl Application {
         let mut stdout = stdout();
 
         execute!(stdout, terminal::EnterAlternateScreen)?;
+        if !self.config.terminal.disable_mouse {
```

Comment:
Let's just call it mouse and have it default to true (https://serde.rs/attr-default.html)